Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -92,23 +88,6 @@ public void setUpStep() throws Exception {
 
         addTextReplacementRule(GLOBAL_ENDPOINT, MOCK_URI + "/");
 
-        int retries = 10;
-        boolean created = false;
-        while (retries > 0) {
-            retries--;
-            try {
-                wireMock = new WireMock(MOCK_HOST, wireMockRule.port());
```

Comment:
I remember in reset, wireMock is set to null. So when will it be created after one test case is run?